CMakeLists.txt

```diff
@@ -657,8 +657,10 @@ if(OPENAL_FOUND)
   else()
     message(STATUS "Using static soundtouch from Externals")
     add_subdirectory(Externals/soundtouch)
-    include_directories(Externals)
   endif()
+  message(STATUS "Using static FreeSurround from Externals")
+  add_subdirectory(Externals/FreeSurround)
+  include_directories(Externals)
```

Comment:
Remove this include_directories(), this is a call that adds the include path to ALL the targets.